### ISSUE TITLE
EasyFarm Freezes

### ISSUE THREAD

**Issue by crhetz** [EasyFarm/EasyFarm#196]
Good morning,

The first time I launched EasyFarm I set out to build a test route. When I finished it the route failed to save and then the program shut down. Now, when I try to launch the program again, Windows Explorer  freezes EasyFarm never loads. I noticed in Task Manager a few instances of WMI Provider Host starting when I click on EasyFarm and they keep on restarting after I try to kill the task.

Any help is appreciated.

Thanks!

**Comment by Mykezero** [comment]
Good Morning! Have you run the dependency checker to make sure you have all the required files?
http://www.elitemmonetwork.com/forums/viewtopic.php?f=28&t=329

Also, did a window pop up with a message indicating what went wrong?
![image](https://cloud.githubusercontent.com/assets/5349608/13780331/5ce39fc2-ea95-11e5-82ba-2a03fe5329cc.png)

I'm not 100% sure what WMI Provider Host does, but from a little bit of research, it looks like a process that monitors the health of other applications. Those instances may have spun up when EF crashed to see what was happening. 

Very strange that Windows Explorer would freeze when launching EF. I would try restarting the computer, and then test EF again to see if things clear up.

**Comment by crhetz** [comment]
Thanks for the response.

I did run the dependency checker and I got the OK for everything. I didn't receive that error, it just said "Failed to save route" at the bottom.

I have tried restarting and relaunching but every time the same thing happens. I double click on the exe and it hangs. I did a system restore and tried again and it worked, but definitely strange.

Thanks again for such a fast response!
